Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1345,5 +1370,195 @@ public static bool SpawnObjects()
             }
             return true;
         }
+
+        // interest management /////////////////////////////////////////////////
+        // Helper function to add all server connections as observers.
+        // This is used if none of the components provides their own
+        // OnRebuildObservers function.
+        internal static void AddAllAuthenticatedConnectionsToObservers(NetworkIdentity identity)
+        {
+            // add all server connections
+            foreach (NetworkConnectionToClient conn in connections.Values)
+            {
+                // only if authenticated (don't send to people during logins)
+                if (conn.isAuthenticated)
+                    identity.AddObserver(conn);
+            }
+
+            // add local host connection (if any)
+            if (localConnection != null)
+            {
+                identity.AddObserver(localConnection);
+            }
+        }
+
+        // allocate newObservers helper HashSet only once
+        static readonly HashSet<NetworkConnection> newObservers = new HashSet<NetworkConnection>();
+
+        // rebuild observers default method (no AOI) - adds all connections
+        static void RebuildObserversDefault(NetworkIdentity identity, bool initialize)
+        {
+            // only add all connections when rebuilding the first time.
+            // second time we just keep them without rebuilding anything.
+            if (initialize)
+            {
+                // not force hidden?
+                if (!identity.forceHidden)
```

Comment:
```suggestion
            if (initialize && !identity.forceHidden)
```

Reply:
same as the other suggestion: it's shorter but in this case I feel like it's easier for the brain if it's spelled out the long way :)